jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TimController.java

```diff
@@ -55,15 +55,19 @@
 
 @Controller
 public class TimController {
-   
-   private static final String ERRSTR = "error";
 
-   private static Logger logger = LoggerFactory.getLogger(TimController.class);
+   private static final Logger logger = LoggerFactory.getLogger(TimController.class);
+
+   private static final String ERRSTR = "error";
+   private static final int THREADPOOL_MULTIPLIER = 3; // multiplier of threads
+                                                       // needed
 
    private OdeProperties odeProperties;
    private DdsDepositor<DdsStatusMessage> depositor;
    private MessageProducer<String, OdeObject> messageProducer;
 
+   private final ExecutorService threadPool;
```

Comment:
Good thought. Since Spring Controllers are singleton objects, we don't need to define the threadpool as static.